rebar.config

```diff
@@ -31,24 +31,24 @@
 %% git config --global url."git://github".insteadOf https://github
 {deps,
  [
-  {base16, ".*", {git, "https://github.com/goj/base16.git", "f78918e"}},
+  {base16, ".*", {git, "https://github.com/goj/base16.git", {ref, "f78918e"}}},
   {exml, ".*", {git, "https://github.com/esl/exml.git", {tag, "3.0.1"}}},
-  {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {ref, "06f676a"}}},
-  {lager_syslog, ".*", {git, "https://github.com/erlang-lager/lager_syslog.git", "3.0.3"}},
-  {cowboy, ".*", {git, "https://github.com/ninenines/cowboy.git", "1.1.2"}},
+  {lager, "3.6.4"},
+  {lager_syslog, ".*", {git, "https://github.com/erlang-lager/lager_syslog.git", {tag, "3.0.3"}}},
```

Comment:
Where we're keeping git dependencies, how about we move from legacy syntax with version regex? I.E. just remove `".*"` part of the tuple everywhere.

Reply:
Good idea, do you want this changed in this PR?

Reply:
I do, I think it fits with the deps upgrade.